pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: layout.pug
+navigationTitle: Upgrade Kommander
+title: Upgrade Kommander
+menuWeight: 20
+excerpt: Steps to upgrade Kommander via CLI
+beta: false
+enterprise: true
+---
+ 
+This section automatically upgrades your Kommander cluster and all its Platform Applications to the latest versions. Kommander is installed on your Management cluster and upgrading it must be the first step you take when upgrading DKP, as it will prevent compatibility issues.
+
+<p class="message--note"><strong>NOTE: </strong>It is important that you upgrade Kommander before upgrading Konvoy and Kubernetes. Kommander 2.1 does not support Kubernetes 1.22, and upgrading Kubernetes to 1.22 will be part of the Konvoy upgrade.</p>
+
+## Prerequisites
+
+- [Download][download-binary] and install the latest DKP CLI binary on your computer.
+- Ensure you are on DKP version 2.1 or 2.1.1 and Kubernetes version 1.21.
+- If you have attached clusters, ensure they are on Kubernetes versions 1.19, 1.20 or 1.21. To upgrade your Kubernetes version, refer to [AKS][AKS], [AWS][AWS], [Azure][Azure], [EKS][EKS], [pre-provisioned][pre-provisioned] documentation. [enterprise type="inline" size="small"/]
+- Review the [Platform Application version updates][platform-apps] that are part of this upgrade.
+
+## Upgrade Kommander
```

Comment:
Should there be an additional step here to have the user make sure their KUBECONFIG environment variable is pointing at the Management cluster?